go.mod

```diff
@@ -14,6 +14,8 @@ require (
 	golang.org/x/crypto v0.1.0
 )
 
+replace github.com/centrifuge/go-substrate-rpc-client/v4 => github.com/Cerebellum-Network/go-substrate-rpc-client/v4 v4.0.9-cere
```

Comment:
Why do you need this replace?

Reply:
It gives less manual work: 
  1. You don't need to replace it in submodules.  
  2. There won't be any conflicts in the code in a case we want to merge something from upstream.

Reply:
But it gives more hidden redirects. Let's get rid of it.